fiche.c

```diff
@@ -668,7 +668,7 @@ static void *handle_connection(void *args) {
     // Write a response to the user
     {
         // Create an url (additional byte for slash and one for new line)
-        const size_t len = strlen(c->settings->domain) + strlen(slug) + 3;
+        const size_t len = strlen(c->settings->domain) + strlen(slug) + 2;
```

Comment:
This fixes a bug in the original code, where a byte was added for a newline but was never used. This caused the last byte of the output to be undefined.